components/applicationsets/kustomization.yaml

```diff
@@ -5,4 +5,6 @@ kind: Kustomization
 resources:
   - core-components-appset.yaml
   # - clusters-appset.yaml
-  #- worker-nodes-appset.yaml
+  # - worker-nodes-appset.yaml
```

Comment:
I don't see an issue with have these uncommented here as this branch is expressly for deploying the cluster.